### DISCUSSION TITLE
BFV/BGV Neg function

### DISCUSSION THREAD

**Discussion by hosseinabdinf** [tuneinsight/lattigo#440] [Q&A] [answered]
Hi, 

I realized that there are no more **Neg** and **NegNew** evaluation functions in Lattigo v5, I could only find the Neg for **Ring**. Should I reimplement them if I need to negate RLWE ciphers?

Thanks in advance

**Comment by Pro7ech** [comment, marked as answer]
Hi @hosseinabdinf, **Neg** has been removed because it is available through **Mul(ct, -1, ct)**
